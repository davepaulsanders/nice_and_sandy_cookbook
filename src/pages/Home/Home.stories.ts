import type { Meta, StoryObj } from '@storybook/react';
 
import Home from './Home';
 
const meta: Meta<typeof Home> = {
  component: Home,
};
 
export default meta;
type Story = StoryObj<typeof Home>;
export const Card: Story = {args: {cardContent: {label: "TikTok Pasta", href: "https://www.delish.com/cooking/recipe-ideas/a35421563/baked-feta-pasta-tiktok/"}}}
