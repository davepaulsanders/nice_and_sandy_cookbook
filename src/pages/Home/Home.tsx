const Home = () => {

return (
   <main className="flex flex-col justify-content-center items-center">
	   <h1 className="text-4xl font-bold text-lgg">Welcome</h1>
	   <h2 className="text-2xl mb-4 text-lightg">to the Nice & Sandy Cookbook!</h2>
	   <p className="text-medg">Categories are listed below:</p>
   </main>
)	
}
export default Home
