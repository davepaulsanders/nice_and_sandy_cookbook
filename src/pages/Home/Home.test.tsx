import Home from "./Home" 
import { expect, test } from 'vitest'
import {render, screen} from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import '@testing-library/dom'

test('<Home /> renders', () => {
	const { container } = render(<Home />)
	expect(container).toMatchSnapshot()
})
