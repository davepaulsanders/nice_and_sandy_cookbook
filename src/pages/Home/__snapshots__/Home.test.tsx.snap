// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Home /> renders 1`] = `
<div>
  <main
    class="flex flex-col justify-content-center items-center"
  >
    <h1
      class="text-4xl font-bold text-lgg"
    >
      Welcome
    </h1>
    <h2
      class="text-2xl mb-4 text-lightg"
    >
      to the Nice & Sandy Cookbook!
    </h2>
    <p
      class="text-medg"
    >
      Categories are listed below:
    </p>
  </main>
</div>
`;
